src/lib/git/push_branch.ts

```diff
@@ -5,12 +5,21 @@ export function pushBranch(opts: {
   remote: string;
   branchName: string;
   noVerify: boolean;
+  forcePush: boolean | false;
 }): void {
+  let forceOption;
+
+  if (opts.forcePush) {
+    forceOption = '--force';
+  } else {
+    forceOption = '--force-with-lease';
+  }
```

Comment:
very slightly more than a nit: avoid `let` when possible 
```suggestion
const forceOption = opts.forcePush ? '--force' : '--force-with-lease';
 ```